src/plugins/intel_cpu/src/nodes/subgraph.h

```diff
@@ -89,6 +114,8 @@ class Snippet : public Node {
     std::vector<int64_t> sch_offsets_in = {};
     std::vector<int64_t> sch_offsets_out = {};
     bool canUseOptimizedImpl = true;
+    // memory buffer for physical broadcasting in dynamic case, use std::vector to facilitate memory management
+    std::vector<float> scratchpad_memory_chunk = {};
```

Comment:
It might be a good idea to use non-continuous memory to improve caching performance (avoid cache line invalidation caused by other threads writes). may change to std::list<std::vector<float>>, but there is still no guarantees about memory locations. Do you think it worth to try?